packages/cozy-harvest-lib/src/models/ConnectionFlow.js

```diff
@@ -904,7 +902,7 @@ export class ConnectionFlow {
     const running =
       trigger?.current_state?.status === 'running' ||
       ![ERRORED, IDLE, SUCCESS].includes(status)
-    const konnectorPolicy = this.getKonnectorPolicy()
+    const konnectorPolicy = this.konnector ? this.getKonnectorPolicy() : null
```

Comment:
We have everywhere in this file call to `this.konnector`. How is it possible to be undefined? 

Does it mean that https://github.com/cozy/cozy-libs/blob/eed38566c7fe2eed1d4e83083d0a97538d27eb23/packages/cozy-harvest-lib/src/models/ConnectionFlow.js#L176 can be undefined here? Then the jsdoc is not right. 

But is is really intended to have a connectionFlow without a konnector? 🤔

Reply:
No, I added a new commit to make the konnector mandatory to FlowProvider/ConnectionFlow. I had to change some components having the FlowProvider as dependency/subdependency and some unit tests. I'm wondering if this should be in another PR